complearn-genbank/routes/redis.js

```diff
@@ -0,0 +1,65 @@
+const redis = require('redis');
+const express = require('express');
+
+const router = express.Router();
+const client = redis.createClient({
+    url: 'redis://localhost:6379',
+    password: 'complearn'
+});
+
+client.connect().catch(console.error);
+
+console.log('Redis routes being registered');
+
+router.post("/get", async (req, res) => {
+    try {
```

Comment:
could change to another name for clearer meaning, here we are /get but passing post body